rllib/agents/dqn/distributional_q_tf_model.py

```diff
@@ -19,76 +19,89 @@ class DistributionalQModel(TFModelV2):
     Note that this class by itself is not a valid model unless you
     implement forward() in a subclass."""
 
-    def __init__(self,
-                 obs_space,
-                 action_space,
-                 num_outputs,
-                 model_config,
-                 name,
-                 q_hiddens=(256, ),
-                 dueling=False,
-                 num_atoms=1,
-                 use_noisy=False,
-                 v_min=-10.0,
-                 v_max=10.0,
-                 sigma0=0.5,
-                 parameter_noise=False):
+    def __init__(
+            self,
+            obs_space,
+            action_space,
+            num_outputs,
+            model_config,
+            name,
+            dueling_hiddens=(256, ),
```

Comment:
Now that I'm thinking about it: "q_head_hiddens" is probably clearer here, as these do not need to be necessarily dueling branches (if dueling == False).